src/components/__stories__/login_header.tsx

```diff
@@ -5,5 +5,5 @@ import LoginHeader from "../login_header"
 
 storiesOf("LoginHeader", LoginHeader)
   .add("Simple login header", () => (
-    <LoginHeader/>
+    <LoginHeader title={"Welcome back, please log in to your account"}/>
```

Comment:
@xtina-starr Ahhh, hmm, then maybe I prematurely merged after all. Basically you should not need to use `Title` inside `LoginHeader` at all, just use the passed in children (`this.props.children`). i.e. in the example that @yuki24 originally posted, the children would be a list consisting of `<Title>Welcome back, please log in to your account</Title>`